version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.5.0"
+	Version = "0.6.0"
```

Comment:
Normally, I'd do this in the same commit as adding the upstream file to `deploy/resources` but I don't guess it matters.

Reply:
I am a fan of maven-release-plugin :)   which bumps to the next version (with `-SNAPSHOT`) _after_ the actual releae is out - so the "old" (previous) release version string is no longer messing around w/ the "master" code